package.json

```
@@ -1,7 +1,7 @@
 {
   "name": "@nuxtjs/web-vitals",
-  "version": "0.1.8",
-  "description": "Web Vitals for Nuxt.js",
+  "version": "0.2.0",
```

(will be bumped with release script)
```suggestion
  "version": "0.1.8",
```